src/components/shared/confirm-modal/ConfirmModal.vue

```diff
@@ -20,7 +20,14 @@
             </div>
             <div class="d-flex justify-content-end">
               <a
-                class="btn btn-sm btn-pink btn-pill mt-4 px-6"
+                class="btn btn-sm btn-pill my-4 px-4 text-pink"
+                href=""
```

Comment:
Do we need an anchor if it's handled with a method?

Reply:
It is a old code. Didn't touched but I'll remove it.